src/main/java/net/snowflake/ingest/utils/HttpUtil.java

```diff
@@ -42,9 +44,27 @@ public class HttpUtil {
   private static String PROXY_SCHEME = "http";
   private static int MAX_RETRIES = 3;
 
-  private static HttpClient httpClient;
+  private static CloseableHttpClient httpClient;
 
-  public static HttpClient getHttpClient() {
+  private static PoolingHttpClientConnectionManager connectionManager;
+
+  private static final int DEFAULT_CONNECTION_TIMEOUT_MINUTES = 1;
+  private static final int DEFAULT_HTTP_CLIENT_SOCKET_TIMEOUT_MINUTES = 5;
```

Comment:
In the log, we don't see any issue that is HTTP connection related, should we just use the old value?